modules/recommended-requirements-central-services.adoc

```diff
@@ -52,6 +53,38 @@ The following table lists the minimum memory and CPU values required to run Cent
 | 16 GiB
 |===
 
+[id="recommended-requirements-central-db-services-central_{context}"]
+== Central-db
```

Comment:
nitpick: For consistency, let's use `Central DB`. (also in the text below)

Reply:
Done